### PR TITLE
adjust apply workflow ordering for errors

### PR DESCRIPTION
**Pull request by verbanicm**
Bootstrapping error file should come first, otherwise the following steps will fail.

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 locals {
   project_id = "guardian-i-42c69c"
-  name       = "test-change"
+  name       = "test-change-test"
```
Comment (sethvargo) [inline terraform/main.tf:17]: Why this change?

**Review by sethvargo — APPROVED** [review]

File: terraform/main.tf (hunk 1 above)
Comment (verbanicm) [reply terraform/main.tf:17]: this is a "unit" test to force workflows to run the guardian repo.
